js/l10n.js

```diff
@@ -0,0 +1,12 @@
+const ipcRenderer = require('electron').ipcRenderer
+const locale = require('../app/locale')
+
+exports.translation = (token) => {
+  if (ipcRenderer) {
+    // If in the renderer process - ask for translation from main process
+    return ipcRenderer.sendSync('translation', token)
```

Comment:
This is an important one, we should async send the translations to the renderer when the window is created and not on each request for a translated string sync.